src_new/util.js

```diff
@@ -1292,6 +1293,10 @@ exports.getUserIds = function(){
 	return window[CONSTANTS.COMMON.PREBID_NAMESPACE].getUserIds();
```

Comment:
With IH Enabled UserID is included by default in OpenWrap and IH

Reply:
Build time assumptions should not be part of code, it will be better and safer to add a check.  if the function is not present then return some default value and add a log that says "the function is not present, please include userId module". Our code is open-source, better to have this check.

Reply:
Agreed.
This code has not updated here but I've added a check for the same. :)